### DISCUSSION TITLE
Drag & Drop Feature Submitted for Review in v2.16.0 - Your Feedback Needed!

### DISCUSSION THREAD

**Discussion by okaryo** [okaryo/TabTabTab#137] [Announcements]
Hi 👋,

Drag & Drop feature has been implemented in version [2.16.0](https://github.com/okaryo/TabTabTab/releases/tag/2.16.0) and submitted for store review. Once it's released, I encourage all of you to give it a try!

Given that it's a significant update, there might be some unexpected bugs or issues. If you encounter any, I would appreciate it if you could share your feedback. Your input is vital for improving the feature!

Also, there are some aspects not addressed in this release but I plan to work on them soon:

* Moving tabs from within a group to outside or to another group. 🔄
* Sorting tabs within a tab group or pin. 🗂️
* Moving tabs to a different window via Drag & Drop. ↗️

**Comment by FreePDF64** [comment]
Hi,

This implemented feature looks very nice! Thx.

I'm looking forward to this planned feature: Moving tabs from within a group to outside or to another group.

cu
